tortoise/src/main/scala/Compiler.scala

```diff
@@ -164,9 +164,9 @@ object Compiler extends CompilerLike {
     else
       defs.head
     if (commands)
-      handlers.commands(pd.statements, true, !raw)
+      handlers.commands(pd.statements, true, !raw)(compilerFlags = compilerFlags, compilerContext = new CompilerContext(blockLevel = 0, source = wrapped))
     else
-      handlers.reporter(pd.statements.stmts(1).args(0))
+      handlers.reporter(pd.statements.stmts(1).args(0))(compilerFlags = compilerFlags, compilerContext = new CompilerContext(blockLevel = 0, source = wrapped))
```

Comment:
Hey, so I got this non-verbose version working, but I noticed we might be able to simplify things a bit further now.  Since the `SourceWrapper` needed to be instantiated explicitly to avoid `String` implicit collisions, we really can skip the intermediate step and instantiate a `CompilerContext` directly.  Let me know if there are problems with [this version](https://gist.github.com/LaCuneta/c01be61a8a97b1494e8ac6a2afb0086b) I'm not seeing (commented out code is the fixed version).